### PR TITLE
update my evaluation results screen for enhancements

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [My Evaluaton Results - Enhancements Revision 1#368](https://github.com/nerubia/kh360-react/issues/368)

BE PR: [update apis for new my evaluation results screen #353](https://github.com/nerubia/kh360-nodejs/pull/353)

### CODE REVIEW

File: src/components/shared/score-range/score-range.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        {score_rating.display_name}
+      </div>
+      <div className='flex text-sm italic leading-loose ml-5 md:w-[860px]'>
+        {score_rating.evaluee_description}
+      </div>
+    </div>
+  )
+}
+
+export default ScoreRange
```
Comment (jatnerubia) [inline src/components/shared/score-range/score-range.tsx:127]: duplicate export? 🤔

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

**Comment by emmaacha** [comment]
Done ^^

**Review by jatnerubia — APPROVED** [review]
Nice

**Review by chadiegilnerubia — APPROVED** [review]

**Pull request merged**